br/pkg/backup/schema.go

```diff
@@ -46,23 +45,6 @@ type Schemas struct {
 	schemas map[string]*schemaInfo
 }
 
-func newBackupSchemas() *Schemas {
-	return &Schemas{
-		schemas: make(map[string]*schemaInfo),
-	}
-}
-
-func (ss *Schemas) addSchema(
```

Comment:
The whole `Schemas` seems to be unused for us.

Reply:
yes. I deleted the whole file in the following commit.